### ISSUE TITLE
Add example of mouse point

### ISSUE THREAD

**Issue by hirako2000** [WhitestormJS/whitestorm.js#175]
There is an example of the mouse module detecting intersection with a Sphere.
Would be nice to see an example of mouse over points as well.

###### Version:
 - [x] v2.x.x
 - [ ] v1.x.x

###### Issue type:
 - [ ] Bug
 - [x ] Proposal/Enhancement
 - [ ] Question

------

<details> 
<summary> <b>Tested on: </b> </summary>
###### --- Desktop
   - [ ] Chrome
   - [ ] Chrome Canary
   - [ ] Chrome dev-channel
   - [ ] Firefox
   - [ ] Opera
   - [ ] Microsoft IE
   - [ ] Microsoft Edge

###### --- Android
   - [ ] Chrome
   - [ ] Firefox
   - [ ] Opera

###### --- IOS
   - [ ] Chrome
   - [ ] Firefox
   - [ ] Opera

</details>

**Comment by liron667** [comment]
Hacktoberfest TLV shotgun

**Comment by noahcoetsee** [comment]
@hirako2000 I am going to take up this issue, as I made many of the other example tutorials, so I can make another example is well, if you'd like my help.

**Comment by hirako2000** [comment]
Excellent! thanks!

**Comment by zardilior** [comment]
I will take it over, to deliver on 3 weeks top as a project for College, more people should do it. I'll do it as a pull request.

**Comment by noahcoetsee** [comment]
Good idea, sorry about this guys, school jumped me. Literally, I got jumped by a teacher 😆

**Comment by hirako2000** [comment]
thanks guys for trying to help out. Further mouse examples would be very valuable to demonstrate how this module works with a few types of events.

**Comment by zardilior** [comment]
I ll devliver before December 3

**Comment by zardilior** [comment]
4
